tasklist/client/src/common/tasks/details/TaskStateLoadingText.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {Loading} from '@carbon/react';
+import type {UserTask} from '@vzeta/camunda-api-zod-schemas/8.8';
+import {useTranslation} from 'react-i18next';
+import styles from './TaskStateLoadingText.module.scss';
+
+type TaskStateLoadingTextProps = {taskState: UserTask['state']};
```

Comment:
As I mentioned before, the common folder should generic and shouldn't refer a specific version type